rrt_params:
  mode: 1 # cost mode: 0: vanilla, 1: height
  max_extension_length: 1.5
  goal_threshold: 0.5 # under this, consider goal is reached
  path_obstacle_threshold: 0.2 # should be the times of grid_resolution
  backward_motion_sampling_rate: 0.2   # 0.0 ~ 1.0



    # num_samples: 1000
    # allowed_computation_time: 300 # in seconds
    # terminate_if_path: false
